### DISCUSSION TITLE
User defined custom search problem

### DISCUSSION THREAD

**Discussion by chrisv5656** [osmandapp/OsmAnd#11450] [Q&A]
When creating a user defined custom search, the search word always has a space added to the end of the word(s).  For instance, if I set up my search to be for example - Campground, it will set the word as - Camground_.  This causes the search to only find instances where Campground is followed by another word.
I am using a Samsung with android version 7.0.
Thank you for your help!

**Comment by tommycrock** [comment]
Hi. Do you mean when you hit the magnifying glass on the keyboard that
shows where the enter would usually be? I'm not sure if it's osmand or the
keyboard that's doing that. What happens if you don't press that? For me it
searches as I type

**Comment by vshcherb** [comment]
Space means that search is complete, so it should be able to find "Campground" or "Campground XXX" but not "CampgroundBasket" as one word.
